terminatorlib/keybindings.py

```diff
@@ -16,19 +16,23 @@
 
 """Terminator by Chris Jones <cmsj@tenshu.net>
 
-Validator and functions for dealing with Terminator's customisable 
+Validator and functions for dealing with Terminator's customisable
 keyboard shortcuts.
 
 """
 
 import re
-from gi.repository import Gtk, Gdk
+from gi.repository import Gdk
```

Comment:
Why remove GTK here?  Is it not necessary?

Reply:
It's an unused import here.

Reply:
Should we remove the comment too?

Reply:
I'd rather keep code cleanup changes separate from feature or bugfix changes.  It means when I go back theough the history I can tell exactly what changes did what.  This is the point I was trying to get across with my earlier comment.  There's too much going on in this commit.  The stuff may be good to include in another PR, but in a PR that is just supposed to implement a feature, the code should just reflect what is necessary to implement that feature.   I definitely welcome bugfixes and code cleanup PRs, just not mixed in with feature implementation PRs